esm_runscripts/tidy.py

```diff
@@ -120,7 +120,7 @@ def assemble_error_list(config):
     gconfig = config["general"]
     known_methods = ["warn", "kill"]
     stdout = (
-        gconfig["thisrun_scripts_dir"]
+        gconfig["experiment_scripts_dir"]
```

Comment:
This are the only real changes needed for fixing the `error stop` feature. The other changes are already merged into release and I don't know why they are showing up here.